### PR TITLE
Adjust zombienet test resources and logic

### PR DESCRIPTION
**Pull request by sandreim**
One more try to make this test robust from a resource perspective.

### CODE REVIEW

**Comment by sandreim** [comment]
Got 2 succesfull runs, but 3rd was really bad:

image

**Comment by pepoviola** [comment]
_Small update_: Our hunch is that the intermittent failures are due to the performance of the vms disks we currently use, we are going to add a new tier to the cluster with more performing disks to run these tests. We will be doing this work tonight at midnight.
Thx!

**Review by rcny — APPROVED** [review]

File: .gitlab/pipeline/zombienet/polkadot.yml [hunk 1]
```diff
@@ -161,6 +161,8 @@ zombienet-polkadot-functional-0011-async-backing-6-seconds-rate:
 zombienet-polkadot-elastic-scaling-0001-basic-3cores-6s-blocks:
   extends:
     - .zombienet-polkadot-common
+  variables:
+    FORCED_INFRA_INSTANCE: "spot-iops"
```
Comment (sandreim) [inline .gitlab/pipeline/zombienet/polkadot.yml:165]: We should use better naming, something like `high-iops-instance` does more justice

Comment (sandreim) [reply .gitlab/pipeline/zombienet/polkadot.yml:165]: cc @pepoviola

**Review by pepoviola — APPROVED** [review]

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
> Tried many times and it looks stable now. I'd say we should merge it.

I tried a couple of time more and works as expected :)

**Review by alindima — APPROVED** [review]

**Pull request merged**